src/components/Accordion/index.tsx

```diff
@@ -57,7 +57,7 @@ const AnimatedAccordion = styled.div`
   max-height: 0;
   /* This is used to calculate a fixed height for the Accordion since height
      transitions require a fixed height for their end height */
-  max-height: ${(props: { hide: boolean }): string => !props.hide && "1500px"};
+  max-height: ${(props: { hide: boolean }): string => !props.hide && "6000px"};
```

Comment:
This seems more like a bandaid fix over the real problem. Since theres a chance the table grows to the point where 6000px is not enough. Maybe we should explore paginating the table?